packages/vue/src/config-provider/src/props.ts

```
@@ -20,4 +20,5 @@ export interface ConfigProviderProps {
     enable?: boolean
     name?: string
   }
+  theme?: Object | null
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)